### PR TITLE
Update ModelsCommand.php

### PR DESCRIPTION
**Pull request by shopapps**
Fix for PHP7.3 error:  array_merge(): Expected parameter 1 to be an array, null given

### CODE REVIEW

File: src/Console/ModelsCommand.php [hunk 1]
```diff
@@ -82,8 +82,8 @@ public function handle()
         $filename = $this->option('filename');
         $this->write = $this->option('write');
         $this->dirs = array_merge(
-            $this->laravel['config']->get('ide-helper.model_locations'),
-            $this->option('dir')
+            (array) $this->laravel['config']->get('ide-helper.model_locations'),
```
Comment (mfn) [inline src/Console/ModelsCommand.php]: This is the same as https://github.com/barryvdh/laravel-ide-helper/pull/835#discussion_r363038544
- don't cast, but provide a default value

File: src/Console/ModelsCommand.php [hunk 2]
```diff
@@ -82,8 +82,8 @@ public function handle()
...
+            (array) $this->option('dir')
```
Comment (mfn) [inline src/Console/ModelsCommand.php]: Why is this necessary?

It's defined as array so it can't be not an array, see https://github.com/barryvdh/laravel-ide-helper/blob/c0bb9ee1fd623f90c2ba593ab151923eb66c933a/src/Console/ModelsCommand.php#L152

File: src/Console/ModelsCommand.php (hunk 1 above)
Comment (DannyvdSluijs) [reply src/Console/ModelsCommand.php:85]: The reason for this not being an array is because in config it was assigned with a different type. Casting to an array would only hide the fact there is a misconfiguration.
Perhaps it would be smart to introducing a config value object/DTO that is made available from the DI. On creation it should validate the config, this to avoid scattered configuration checks throughout the code.

File: src/Console/ModelsCommand.php (hunk 2 above)
Comment (DannyvdSluijs) [reply src/Console/ModelsCommand.php:86]: Probably the array casting is added to ensure it becomes an array. The signature `InteractsWithIO::option(string $key = null): string|array|bool|null` shows it potentially can return many types. I'm not sure if this is a good thing to add though it would break when changing the definition.

**Comment by mfn** [comment]
@barryvdh I don't think casting the returned value from the config is the correct approach: it would simply hide bugs in the config that way or have other side effects even harder to debug => I vote for closing this PR!
